cmd/helm/install_test.go

```diff
@@ -207,6 +207,12 @@ func TestInstall(t *testing.T) {
 			name: "install chart with only crds",
 			cmd:  "install crd-test testdata/testcharts/chart-with-only-crds --namespace default",
 		},
+		// Install chart with repoURL
+		{
+			name:   "install chart with only repoURL",
+			cmd:    "install traefik traefik --repo https://helm.traefik.io/traefik --version 9.18.0",
```

Comment:
I think this piece should use a mock request, but it looks like it requires a lot of code changes. Is there any suitable solution? thank you
@mattfarina